src/views/standalone/wizard/SshStep.vue

```diff
@@ -0,0 +1,190 @@
+<!--
+  Copyright (C) 2025 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import {
+  NeButton,
+  NeHeading,
+  NeCard,
+  NeTextInput,
+  focusElement,
+  getAxiosErrorMessage,
+  NeToggle,
+  NeInlineNotification
+} from '@nethesis/vue-components'
+import {
+  faArrowRight,
+  faArrowLeft,
+  faCircleCheck,
+  faCircleXmark
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { ref } from 'vue'
+import { MessageBag } from '@/lib/validation'
+import { ubusCall } from '@/lib/standalone/ubus'
+import { useSetupWizardStore } from '@/stores/standalone/setupWizard'
+
+const emit = defineEmits<{
+  nextStep: []
+  previousStep: []
+}>()
+
+const { t } = useI18n()
+const wizardStore = useSetupWizardStore()
+const portRef = ref()
```

Comment:
maybe the usage of `useTemplateRef` is better
https://vuejs.org/guide/typescript/composition-api.html#typing-template-refs